draft-ietf-mls-protocol.md

```diff
@@ -2180,7 +2180,10 @@ not be listed, while any credential types the application wishes to use MUST
 be listed. Extensions that appear in the `extensions` field of a LeafNode
 MUST be included in the `extensions` field of the `capabilities` field, and the
 credential type used in the LeafNode MUST be included in the `credentials` field
-of the `capabilities` field.
+of the `capabilities` field.  As discussed in {{extensibility}}, unknown values
+in `capabilities` MUST be ignored, and the creator of a `capabilities` field
+SHOULD contain some random GREASE values to help ensure that other clients correctly
```

Comment:
This is the first line in the document where GREASE is mentioned. It should be spelled out and have a reference.